packages/block-library/src/cover/test/edit.js

```diff
@@ -0,0 +1,321 @@
+/**
+ * External dependencies
+ */
+import { screen, fireEvent, act, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+/**
+ * Internal dependencies
+ */
+import {
+	initializeEditor,
+	selectBlock,
+} from 'test/integration/helpers/integration-test-editor';
+
+async function setup( attributes ) {
+	const testBlock = { name: 'core/cover', attributes };
+	return initializeEditor( testBlock );
+}
+
+async function createAndselectBlock() {
```

Comment:
```suggestion
async function createAndSelectBlock() {
```

Small typo, all the call sites would need an update too.

Reply:
Thanks, fixed.